test_communication/test/test_messages_c.cpp

```diff
@@ -174,7 +174,7 @@ class CLASSNAME (TestMessagesFixture, RMW_IMPLEMENTATION) : public ::testing::Te
           fini_message(&message);
         });
 
-        rcl_wait_set_t wait_set = rcl_get_zero_initialized_wait_set();
+        wait_set = rcl_get_zero_initialized_wait_set();
```

Comment:
very good point. fixed in https://github.com/ros2/system_tests/pull/189/commits/d444705ab1fab805ec6715b0784e9291f7fcd363